test/core/rules/config_test.py

```diff
@@ -21,8 +24,14 @@ def _eval(self, context):
         pass
 
 
+@document_fix_compatible
 class Rule_T001(BaseRule):
-    """A deliberately malicious rule."""
+    """A deliberately malicious rule.
+
+    **Anti-pattern**
```

Comment:
Had to add this verbiage to the docstring so `@ document_fix_compatible` would work.